app/src/main/java/co/nimblehq/compose/crypto/ui/screens/detail/DetailScreen.kt

```diff
@@ -0,0 +1,185 @@
+                modifier = Modifier
+                    .constrainAs(currentPrice) {
+                        top.linkTo(logo.bottom)
+                        linkTo(start = parent.start, end = parent.end)
+                    }
+                    .padding(vertical = Dp8),
+                // TODO: Remove dummy value when work on Integrate.
+                text = stringResource(R.string.coin_currency, "3,260.62"),
+                color = MaterialTheme.colors.textColor,
+                style = Style.semiBold24()
+            )
+
+            PriceChangeButton(
+                modifier = Modifier
+                    .constrainAs(priceChangePercent) {
+                        top.linkTo(currentPrice.bottom)
+                        linkTo(start = parent.start, end = parent.end)
+                    },
+                iconPaddingEnd = Dp9,
+                backgroundColor = Color.WhiteIce,
+                // TODO: Remove dummy image when work on Integrate.
+                priceChangePercent = "6.21",
+                isPositiveNumber = true
+            )
+
+            // TODO: Update this section when work create UI for a graph.
+            Spacer(
+                modifier = Modifier
+                    .height(350.dp)
+                    .constrainAs(graph) {
+                        top.linkTo(priceChangePercent.bottom)
+                    }
+            )
+
+            CoinInfo(
+                modifier = Modifier.constrainAs(coinInfoItem) {
+                    top.linkTo(graph.bottom)
+                },
+                sellBuyLayoutHeight = sellBuyLayoutHeight.value
+            )
+        }
+
+        Box(
+            modifier = Modifier
+                .fillMaxSize()
+                .navigationBarsPadding(),
+            contentAlignment = Alignment.BottomEnd
+        ) {
+            SellBuyGroup(
+                modifier = Modifier.onGloballyPositioned {
+                    sellBuyLayoutHeight.value = with(localDensity) { it.size.height.toDp() }
+                }
+            )
+        }
+    }
+}
+
+@Suppress("FunctionNaming")
+@Composable
+private fun CoinInfo(
+    modifier: Modifier,
+    sellBuyLayoutHeight: Dp
+) {
+    // TODO: Remove dummy value when work on Integrate.
+    Column(modifier = modifier.padding(start = Dp16, end = Dp16, bottom = sellBuyLayoutHeight)) {
+        DetailItem(
+            modifier = Modifier,
+            title = "Market Cap",
+            price = "387,992,058,833.42",
+            pricePercent = 7.32
+        )
+
+        DetailItem(
+            modifier = Modifier.padding(top = Dp16),
+            title = "All Time High",
+            price = "4,891.70",
+            pricePercent = 33.42
+        )
+
+        DetailItem(
+            modifier = Modifier.padding(vertical = Dp16),
+            title = "All Time Low",
+            price = "0.4209",
+            pricePercent = 773717.23
+        )
+    }
+}
+
+@Suppress("FunctionNaming")
+@Composable
+@Preview
```

Comment:
can we preview this screen with system ui?

```suggestion
@Preview(showSystemUi = true, uiMode = UI_MODE_NIGHT_NO)
```